oci/spec_opts.go

```diff
@@ -1325,11 +1325,19 @@ func WithLinuxDevices(devices []specs.LinuxDevice) SpecOpts {
 }
 
 // WithLinuxDevice adds the device specified by path to the spec
-func WithLinuxDevice(path, permissions string) SpecOpts {
+func WithLinuxDevice(path, permissions string, followSymlinks bool) SpecOpts {
```

Comment:
We need to keep this function's signature backward-compatible at least in 1.x.

Reply:
Makes sense... I can make this a separate `Withxxx` option

Reply:
Perhaps make a non-exported function that does both (don't have a good name for the new one :joy:), like;


```go
func WithLinuxDevice(path, permissions string) SpecOpts {
    return withLinuxDevice(path, permissions, false)
}

fun WithLinuxdeviceFollowSymLinks(path, permissions string) SpecOpts {
    return withLinuxDevice(path, permissions, true)
}


func withLinuxDevice(path, permissions string, followSymlinks bool) SpecOpts {
     ....
}
```

Reply:
Naming is the hard part :) thanks for the suggestion, this is a much cleaner way to accomplish things